packages/preview/dndcampaign/0.1.0/typst.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "dndcampaign"
```

Comment:
Could you please change the name to something less canonical / a bit more unique? Other people might want to submit templates for the same purpose in the future.